pkg/ingester/activeseries/active_series.go

```diff
@@ -210,6 +231,18 @@ func (c *ActiveSeries) ActiveWithMatchers() (total int, totalMatching []int, tot
 	return
 }
 
+func (c *ActiveSeries) ActiveByAttributionValue() map[string]uint32 {
+	total := map[string]uint32{}
```

Comment:
This can also be initialized to max size

Reply:
done